pkg/operator/credential_request.go

```diff
@@ -103,3 +104,56 @@ func addCredentialsRequestHash(cr *unstructured.Unstructured) error {
 	cr.SetAnnotations(annotations)
 	return nil
 }
+
+// getCredentialsRequestFailure finds all true conditions in CredentialsRequest
+// and composes an error message from them.
+func getCredentialsRequestFailure(cr *unstructured.Unstructured) string {
```

Comment:
Did you intend to use this only in the unit test?

Reply:
No! Some code is missing, I reopened https://bugzilla.redhat.com/show_bug.cgi?id=1835726